bin/common.c

```diff
@@ -13,13 +13,15 @@
  * permissions and limitations under the License.
  */
 
+#include "common.h"
 #include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
 #include <getopt.h>
 #include <errno.h>
 #include <s2n.h>
 
+#define PSK_SECRET_SIZE_MAX 2048
```

Comment:
Why do we need a limit on secret size? Are we advertising it somewhere?

Reply:
This size is the maximum size of the psk secret. We are not advertising this value as it is the max size for PSK secrets used in practice. 

```
struct s2n_blob secret = { 0 };
uint8_t secret_buf[PSK_SECRET_SIZE_MAX] = { 0 };
POSIX_GUARD(s2n_blob_init(&secret, secret_buf, sizeof(secret_buf)));
POSIX_GUARD(s2n_hex_string_to_bytes((const uint8_t *)token, &secret));
GUARD_EXIT(s2n_psk_set_secret(psk, secret.data, secret.size), "Error setting psk secret");
```